src/Libraries/Microsoft.Extensions.AI/Functions/AIFunctionFactory.cs

```diff
@@ -38,43 +32,30 @@ public static class AIFunctionFactory
     public static AIFunction Create(Delegate method, AIFunctionFactoryCreateOptions options)
```

Comment:
options should be nullable, yes?

Reply:
Yes can be made nullable, but not optional.